resolwe_bio/processes/workflows/quantseq.py

```diff
@@ -0,0 +1,374 @@
+            input_featurecounts.update(input)
+
+        quantification = Data.create(
+            process=BioProcess.get_latest(slug="feature_counts"),
+            input=input_featurecounts,
+        )
+
+        input_seqtk = {
+            "reads": preprocessing,
+            "n_reads": inputs.downsampling.n_reads,
+            "advanced": {
+                "seed": inputs.downsampling.advanced.seed,
+                "fraction": inputs.downsampling.advanced.fraction,
+                "two_pass": inputs.downsampling.advanced.two_pass,
+            },
+        }
+
+        if inputs.reads.type.startswith("data:reads:fastq:single:"):
+            process_slug = "seqtk-sample-single"
+        elif inputs.reads.type.startswith("data:reads:fastq:paired:"):
+            process_slug = "seqtk-sample-paired"
+        else:
+            self.error("Wrong reads input type was provided.")
+
+        downsampling = Data.create(
+            process=BioProcess.get_latest(slug=process_slug),
+            input=input_seqtk,
+        )
+
+        idxstats = Data.create(
+            process=BioProcess.get_latest(slug="samtools-idxstats"),
+            input={
+                "alignment": alignment,
+            },
+        )
+
+        alignment_qorts = Data.create(
+            process=BioProcess.get_latest(slug="alignment-star"),
+            input={
+                "reads": downsampling,
+                "genome": inputs.genome,
+            },
+        )
+        input_qorts = {
+            "alignment": alignment_qorts,
+            "annotation": inputs.annotation,
+            "options": {
+                "stranded": inputs.assay_type,
+            },
+        }
+
+        if inputs.trimming_tool == "cutadapt":
+            input = {"options": {"stranded": "forward"}}
+            input_qorts.update(input)
+
+        qorts = Data.create(
+            process=BioProcess.get_latest(slug="qorts-qc"),
+            input=input_qorts,
+        )
+
+        multiqc_inputs = [
+            inputs.reads,
+            preprocessing,
+            alignment,
+            downsampling,
+            quantification,
+            idxstats,
+            qorts,
+        ]
+
+        if inputs.rrna_reference and inputs.globin_reference:
+
+            alignment_qc_rrna = Data.create(
+                process=BioProcess.get_latest(slug="alignment-star"),
+                input={
+                    "reads": downsampling,
+                    "genome": inputs.rrna_reference,
+                },
+            )
+            alignment_qc_globin = Data.create(
+                process=BioProcess.get_latest(slug="alignment-star"),
+                input={
+                    "reads": downsampling,
+                    "genome": inputs.globin_reference,
+                },
+            )
+
+            multiqc_inputs.extend([alignment_qc_rrna, alignment_qc_globin])
+
+        inputs_multiqc = {"data": multiqc_inputs}
+
+        if inputs.trimming_tool == "bbduk":
```

Comment:
This block can should be omitted. Both inputs are set to `True` in the `multiqc` process.